src/pages/Home.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react'
+import Todos from '../components/Todos'
+import '../styles/Home.css'
+
+const Home = () => {
+  const [name, setName] = useState("Amir")
+  return (
+    <div className='home'>
+      <h1 className="welcome">
+        Welcome Back, <span className='name'>{name}</span>!
+      </h1>
+      <h3 className='todosleft'>You have {3} tasks for today. All the best!</h3>
```

Comment:
didn't understand the purpose of wrapping 3 in `{}`

Reply:
I forgot to put todos.length there😩. Thanks for pointing it out.